src/sentry/analytics/events/alert_sent.py

```diff
@@ -0,0 +1,22 @@
+from sentry import analytics
+
+
+class AlertSentEvent(analytics.Event):
+    type = "alert.sent"
+
+    attributes = (
+        analytics.Attribute("organization_id"),
+        analytics.Attribute("project_id", required=False),
```

Comment:
@scttcper when would we not have a project_id?

Reply:
i'm not sure, the other notification_sent analytic had it and i started with that

Reply:
@scttcper can we make it required? I think that's better.